<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechercheSatellite</class>
 <widget class="QFrame" name="RechercheSatellite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="satellitesTrouves">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>49</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_satellitesTrouves">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>31</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Objets trouvés :</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_41">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_nomRechercheSat">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nom">
      <property name="font">
       <font/>
      </property>
      <property name="toolTip">
       <string>Entrez au minimum 3 lettres du nom de l'objet</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_nom">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frameResultats">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>46</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget_44">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_RechercheNomsat">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="nomsat">
       <property name="text">
        <string notr="true">ISS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_nomsat">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_45">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_rechercheNorad_2">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_numNorad_2">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Numéro NORAD :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_desigCospar_2">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Désignation COSPAR :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="numNorad">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">25544</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="desigCospar">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">1998-067A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>0</y>
      <width>353</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_rechercheSat1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout_rechercheSat1">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_magnitudeStdMaxDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Magnitude std/max :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_modeleDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Modèle orbital :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_dimensionsDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Dimensions/Section :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="magnitudeStdMaxDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string notr="true">-0.5v/-2.9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="modeleDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string notr="true">SGP4 (NE)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="dimensionsDonneesSat">
         <property name="font">
          <font/>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string notr="true">Cylindrique. L=30.0m, R=20.0m / 350.00 m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QLabel" name="sqDonneesSat">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_46">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>191</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_rechercheSat2">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_dateLancementDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Date de lancement :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_dateRentree">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Date de rentrée :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_categorieOrbiteDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Catégorie d'orbite :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_paysDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Pays/Organisation :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_siteLancementDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Site de lancement :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="dateLancementDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">2013/07/08</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="dateRentree">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">2013/07/08</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="categorieOrbiteDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">LEO/I</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="paysDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">US</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="siteLancementDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">AFETR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_47">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>55</y>
      <width>246</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_rechercheSat3">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_apogeeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Apogée (Altitude) :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_perigeeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Périgée (Altitude) :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_periodeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Période orbitale :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_inclinaisonDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Inclinaison :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="apogeeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">036000.0 km (036000.0 km)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="perigeeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">36000.0 km (36000.0 km)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="periodeDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">24h 50m 53s</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="inclinaisonDonneesSat">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">000.0000°</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_59">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_rechercheSat4">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_fichiersElementsOrbitaux">
       <property name="text">
        <string>Fichiers :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="fichiersElementsOrbitaux">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_43">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_cosparRechercheSat">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="cosparDonneesSat">
      <property name="font">
       <font/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_cosparDonneesSat">
      <property name="text">
       <string>COSPAR :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_42">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>0</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_noradRechercheSat">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QSpinBox" name="noradDonneesSat">
      <property name="font">
       <font/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_noradDonneesSat">
      <property name="text">
       <string>NORAD :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
